### PR TITLE
NIAD-1610: handle diagnostic report practitioners

### PR DESCRIPTION
**Pull request by ElliotB-96**

### CODE REVIEW

File: service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java [hunk 1]
```diff
@@ -211,4 +227,35 @@ private String buildNarrativeStatementForDiagnosticReport(DiagnosticReport diagn
             narrativeStatementTemplateParameters.build()
         );
     }
+
+    private String buildListOfHumanReadableNames(DiagnosticReport diagnosticReport) {
```
Comment (bartek-sarul) [inline service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java:231]: If this function operates only on `List<DiagnosticReport.DiagnosticReportPerformerComponent>` then this should be the argument of this function

Comment (bartek-sarul) [reply service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java:231]: The same thing in `buildNarrativeStatementForDiagnosticReport` where it uses `diagnosticReport.getIssuedElement()`

**Review by bartek-sarul — CHANGES_REQUESTED** [review]

File: service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java [hunk 2]
```diff
@@ -179,36 +185,78 @@ private String prepareReportLevelNarrativeStatements(DiagnosticReport diagnostic
...
 
         return TemplateUtils.fillTemplate(
             NARRATIVE_STATEMENT_TEMPLATE,
             narrativeStatementTemplateParameters.build()
         );
     }
+
+    private String buildListOfHumanReadableNames(List<DiagnosticReport.DiagnosticReportPerformerComponent> performers) {
+        return
+            performers.stream()
```
Comment (bartek-sarul) [inline service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java:235]: A bit strange formatting

Comment (ElliotB-96) [reply service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java]: Not sure what happened there, will fix it

**Pull request merged**